src/authentication/src/steps/SelectServer.jsx

```diff
@@ -60,6 +61,22 @@ export class SelectServer extends Component {
   }
 
   onChange(value) {
+    if (value.includes('www.') && (value.match(/\./g) || []).length === 1) {
```

Comment:
This line is hard to decipher, I'm still not sure what we're testing for?

Reply:
If an user puts a dot, it's mainly because this user has a custom domain (since we can't have any dot in our cozy url). But we also accept the user writes `www.ylohse` and in this case we want to use our `mycozy.cloud` domain.